homeassistant/components/isy994/binary_sensor.py

```diff
@@ -216,57 +357,53 @@ def value(self) -> object:
             # Do this first so we don't invert None on moisture sensors
             return None
 
-        if self.device_class == "moisture":
+        if self.device_class == DEVICE_CLASS_MOISTURE:
             return not self._computed_state
 
         return self._computed_state
 
-    @property
-    def is_on(self) -> bool:
-        """Get whether the ISY994 binary sensor device is on.
-
-        Note: This method will return false if the current state is UNKNOWN
-        """
-        return bool(self.value)
-
     @property
     def state(self):
         """Return the state of the binary sensor."""
         if self._computed_state is None:
-            return None
+            return STATE_UNKNOWN
```

Comment:
We use `None` to represent unknown state.

Reply:
OK, I was miseled by some other integrations and thought None was outdated. Will revert this and other locations in the next PR. Thanks!